stylelint-plugin/rules/use-proper-token/token-map.js

```diff
@@ -1,43 +1,43 @@
 /**
  * Map of CSS properties and their corresponding allowed design tokens
- * Example: `background,background-color` CSS properties allowed tokens are `color-background` and `method-color-background`
+ * Example: for `background-color` CSS property allowed tokens are `color-background` and `method-color-background`
```

Comment:
Can you expand the comment here a bit more to say something like "as long as component tokens follow X pattern, the component tokens do not need to be added to the arrays below"

Reply:
Done